src/demos/building-a-geospatial-app/1-starting-with-map/src/app.js

```diff
@@ -51,7 +51,7 @@ export default class App extends Component {
         <MapGL
           {...this.state.viewport}
           mapStyle={MAPBOX_STYLE}
-          onViewportChange={viewport => this._onViewportChange.bind(viewport)}
+          onViewportChange={viewport => this._onViewportChange(viewport)}
```

Comment:
I would be in favor of class properties, don't think the new form would differ that much for newcomers to the current one, and would avoid context issues. Thoughts?

Reply:
Yeah I was wondering that. On one hand we don't want to introduce too much overhead. On the other hand the babel plugins are already set up, and less visual clutter is definitely a goal here. I'm in favor of changing.

This particular example is just fixing a bug though.

Reply:
#33